test_data/curried_fn.good.nix

```diff
@@ -4,5 +4,5 @@
       err = t: v: abort
         "oups";
     in
-    92;
+      92;
```

Comment:
in a previous discussion we argues that the body should not be indented, similarly to the other keywords like `asset` and `with`. I don't have a strong preference though.

Reply:
Yeah, I sort of came to liking the indentation here, the in keyword makes it feel different that with

Reply:
I prefer the new version.

Reply:
The new version is closer to the format used predominantly in nixpkgs as well